src/custom/state/orders/consts.ts

```diff
@@ -7,3 +8,6 @@ export const ContractDeploymentBlocks: Partial<Record<ChainId, number>> = {
 }
 
 export const OPERATOR_API_POLL_INTERVAL = 10000 // in ms
+export const PENDING_ORDERS_PRICE_CHECK_POLL_INTERVAL = 30000 // in ms
+
+export const OUT_OF_MARKET_PRICE_DELTA_PERCENTAGE = new Percent(1, 100) // 1/100 => 0.01 => 1%
```

Comment:
nit: `1%` as a comment is clear enough 😂

Reply:
Well, I like to be extra clear ;)

Also, there are other places where I saw Percent(1, 10000) and I confess I have to think awhile to understand what it does.

Reply:
Hehe, I like the extra clarity :)